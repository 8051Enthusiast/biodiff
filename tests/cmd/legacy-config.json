{"algo":{"gap_open":-5,"gap_extend":-1,"mismatch_score":-1,"match_score":1,"mode":{"Blockwise":8192},"band":"Normal"},"style":{"mode":"HexAsciiMix","ascii_col":false,"bars_col":true,"vertical":false,"spacer":true,"right_to_left":false}}