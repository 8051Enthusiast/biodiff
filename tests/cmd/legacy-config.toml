bin.name = "biodiff"
args = ["--color", "--config", "tests/cmd/legacy-config.json", "--print", "tests/text_x.txt", "tests/text_y.txt"]
status.code = 0