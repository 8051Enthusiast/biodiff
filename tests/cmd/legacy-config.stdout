[38;5;8m 0000 [39mＴ ｈ ｉ ｓ ﹍ ｉ ｓ ﹍  ｓ ｏ ｍ ｅ ﹍ ｃ ｏ ｍ [38;5;8m |[39m▃▄▄▄▁▄▄▁▄▄▄▄▁▄▄▄[38;5;8m | 0000 [39mＴ ｈ ｉ ｓ ﹍ ｉ ｓ ﹍  ｓ ｏ ｍ ｅ ﹍ ｃ ｏ ｍ [38;5;8m |[39m▃▄▄▄▁▄▄▁▄▄▄▄▁▄▄▄
[38;5;8m 0010 [39mｍ ｏ ｎ ﹍ ｔ ｅ ｘ ｔ  ﹍ ｂ ｅ ｔ ｗ ｅ ｅ ｎ [38;5;8m |[39m▄▄▄▁▄▄▄▄▁▄▄▄▄▄▄▄[38;5;8m | 0010 [39mｍ ｏ ｎ ﹍ ｔ ｅ ｘ ｔ  ﹍ ｂ ｅ ｔ ｗ ｅ ｅ ｎ [38;5;8m |[39m▄▄▄▁▄▄▄▄▁▄▄▄▄▄▄▄
[38;5;8m 0020 [39m﹍ ｂ ｏ ｔ ｈ ﹍ ｆ ｉ  ｌ ｅ ｓ [38;5;10m． ﹍ Ｔ ｈ ｉ [38;5;8m |[39m▁▄▄▄▄▁▄▄▄▄▄[38;5;10m▂▁▃▄▄[38;5;8m | 0020 [39m﹍ ｂ ｏ ｔ ｈ ﹍ ｆ ｉ  ｌ ｅ ｓ [38;5;2m               [38;5;8m |[39m▁▄▄▄▄▁▄▄▄▄▄[38;5;2m░░░░░[39m
[38;5;8m 0030 [38;5;10mｓ ﹍ ｔ ｅ ｘ ｔ ﹍ ｉ  ｓ ﹍ ｏ ｎ ｌ ｙ ﹍ ｉ [38;5;8m |[38;5;10m▄▁▄▄▄▄▁▄▄▁▄▄▄▄▁▄[38;5;8m | 002b [38;5;2m                                                 [38;5;8m |[38;5;2m░░░░░░░░░░░░░░░░[39m
[38;5;8m 0040 [38;5;10mｎ ﹍ ｔ ｈ ｉ ｓ ﹍ ｆ  ｉ ｒ ｓ ｔ ﹍ ｆ ｉ ｌ [38;5;8m |[38;5;10m▄▁▄▄▄▄▁▄▄▄▄▄▁▄▄▄[38;5;8m | 002b [38;5;2m                                                 [38;5;8m |[38;5;2m░░░░░░░░░░░░░░░░[39m
[38;5;8m 0050 [38;5;10mｅ [39m． /n Ｔ ｈ ｅ ﹍ ｃ  ｈ ａ ｒ ａ ｃ ｔ ｅ ｒ [38;5;8m |[38;5;10m▄[39m▂▁▃▄▄▁▄▄▄▄▄▄▄▄▄[38;5;8m | 002b [38;5;2m   [39m． /n Ｔ ｈ ｅ ﹍ ｃ  ｈ ａ ｒ ａ ｃ ｔ ｅ ｒ [38;5;8m |[38;5;2m░[39m▂▁▃▄▄▁▄▄▄▄▄▄▄▄▄
[38;5;8m 0060 [39m﹍ ｉ ｓ ﹍ [38;5;9mＸ [39m﹍ ｉ ｎ  ｓ ｔ ｅ ａ ｄ ﹍ ｏ ｆ [38;5;8m |[39m▁▄▄▁[38;5;9m▃[39m▁▄▄▄▄▄▄▄▁▄▄[38;5;8m | 003a [39m﹍ ｉ ｓ ﹍ [38;5;9mＹ [39m﹍ ｉ ｎ  ｓ ｔ ｅ ａ ｄ ﹍ ｏ ｆ [38;5;8m |[39m▁▄▄▁[38;5;9m▃[39m▁▄▄▄▄▄▄▄▁▄▄
[38;5;8m 0070 [39m﹍ [38;5;9mＹ [39m． /n Ｓ ｏ ｍ ｅ  ﹍ ｍ ｏ ｒ ｅ ﹍ ｔ ｅ [38;5;8m |[39m▁[38;5;9m▃[39m▂▁▃▄▄▄▁▄▄▄▄▁▄▄[38;5;8m | 004a [39m﹍ [38;5;9mＸ [39m． /n Ｓ ｏ ｍ ｅ  ﹍ ｍ ｏ ｒ ｅ ﹍ ｔ ｅ [38;5;8m |[39m▁[38;5;9m▃[39m▂▁▃▄▄▄▁▄▄▄▄▁▄▄
[38;5;8m 0080 [39mｘ ｔ ﹍ ｔ ｏ ﹍ ｍ ａ  ｋ ｅ ﹍ ｓ ｕ ｒ ｅ ﹍ [38;5;8m |[39m▄▄▁▄▄▁▄▄▄▄▁▄▄▄▄▁[38;5;8m | 005a [39mｘ ｔ ﹍ ｔ ｏ ﹍ ｍ ａ  ｋ ｅ ﹍ ｓ ｕ ｒ ｅ ﹍ [38;5;8m |[39m▄▄▁▄▄▁▄▄▄▄▁▄▄▄▄▁
[38;5;8m 0090 [39mｔ ｈ ｅ ﹍ ｎ ｕ ｍ ｂ  ｅ ｒ ﹍ ｏ ｆ ﹍ ｓ ａ [38;5;8m |[39m▄▄▄▁▄▄▄▄▄▄▁▄▄▁▄▄[38;5;8m | 006a [39mｔ ｈ ｅ ﹍ ｎ ｕ ｍ ｂ  ｅ ｒ ﹍ ｏ ｆ ﹍ ｓ ａ [38;5;8m |[39m▄▄▄▁▄▄▄▄▄▄▁▄▄▁▄▄
[38;5;8m 00a0 [39mｍ ｅ ﹍ ｃ ｈ ａ ｒ ａ  ｃ ｔ ｅ ｒ ｓ ﹍ ｂ ｅ [38;5;8m |[39m▄▄▁▄▄▄▄▄▄▄▄▄▄▁▄▄[38;5;8m | 007a [39mｍ ｅ ﹍ ｃ ｈ ａ ｒ ａ  ｃ ｔ ｅ ｒ ｓ ﹍ ｂ ｅ [38;5;8m |[39m▄▄▁▄▄▄▄▄▄▄▄▄▄▁▄▄
[38;5;8m 00b0 [39mｔ ｗ ｅ ｅ ｎ ﹍ ｔ ｈ  ｅ ﹍ ｔ ｅ ｘ ｔ ｓ ﹍ [38;5;8m |[39m▄▄▄▄▄▁▄▄▄▁▄▄▄▄▄▁[38;5;8m | 008a [39mｔ ｗ ｅ ｅ ｎ ﹍ ｔ ｈ  ｅ ﹍ ｔ ｅ ｘ ｔ ｓ ﹍ [38;5;8m |[39m▄▄▄▄▄▁▄▄▄▁▄▄▄▄▄▁
[38;5;8m 00c0 [39mｉ ｓ ﹍ ｓ ｕ ｆ ｆ ｉ  ｃ ｉ ｅ ｎ ｔ ｌ ｙ ﹍ [38;5;8m |[39m▄▄▁▄▄▄▄▄▄▄▄▄▄▄▄▁[38;5;8m | 009a [39mｉ ｓ ﹍ ｓ ｕ ｆ ｆ ｉ  ｃ ｉ ｅ ｎ ｔ ｌ ｙ ﹍ [38;5;8m |[39m▄▄▁▄▄▄▄▄▄▄▄▄▄▄▄▁
[38;5;8m 00d0 [39mｈ ｉ ｇ ｈ ． /n [38;5;2m                               [38;5;8m |[39m▄▄▄▄▂▁[38;5;2m░░░░░░░░░░[38;5;8m | 00aa [39mｈ ｉ ｇ ｈ ． /n [38;5;10mＴ ｈ  ｉ ｓ ﹍ ｔ ｅ ｘ ｔ ﹍ [38;5;8m |[39m▄▄▄▄▂▁[38;5;10m▃▄▄▄▁▄▄▄▄▁[39m
[38;5;8m 00d6 [38;5;2m                                                 [38;5;8m |[38;5;2m░░░░░░░░░░░░░░░░[38;5;8m | 00ba [38;5;10mｉ ｓ ﹍ ｏ ｎ ｌ ｙ ﹍  ｉ ｎ ﹍ ｔ ｈ ｉ ｓ ﹍ [38;5;8m |[38;5;10m▄▄▁▄▄▄▄▁▄▄▁▄▄▄▄▁[39m
[38;5;8m 00d6 [38;5;2m                                        [39mＨ ｅ ｒ [38;5;8m |[38;5;2m░░░░░░░░░░░░░[39m▃▄▄[38;5;8m | 00ca [38;5;10mｓ ｅ ｃ ｏ ｎ ｄ ﹍ ｆ  ｉ ｌ ｅ ． ﹍ [39mＨ ｅ ｒ [38;5;8m |[38;5;10m▄▄▄▄▄▄▁▄▄▄▄▂▁[39m▃▄▄
[38;5;8m 00d9 [39mｅ ﹍ ｉ ｓ ﹍ ｓ ｏ ｍ  ｅ ﹍ ｍ ｏ ｒ ｅ ﹍ ｃ [38;5;8m |[39m▄▁▄▄▁▄▄▄▄▁▄▄▄▄▁▄[38;5;8m | 00da [39mｅ ﹍ ｉ ｓ ﹍ ｓ ｏ ｍ  ｅ ﹍ ｍ ｏ ｒ ｅ ﹍ ｃ [38;5;8m |[39m▄▁▄▄▁▄▄▄▄▁▄▄▄▄▁▄
[38;5;8m 00e9 [39mｏ ｍ ｍ ｏ ｎ ﹍ ｔ ｅ  ｘ ｔ ﹍ ｔ ｏ ﹍ ｗ ｒ [38;5;8m |[39m▄▄▄▄▄▁▄▄▄▄▁▄▄▁▄▄[38;5;8m | 00ea [39mｏ ｍ ｍ ｏ ｎ ﹍ ｔ ｅ  ｘ ｔ ﹍ ｔ ｏ ﹍ ｗ ｒ [38;5;8m |[39m▄▄▄▄▄▁▄▄▄▄▁▄▄▁▄▄
[38;5;8m 00f9 [39mａ ｐ ﹍ ｉ ｔ ﹍ ｕ ｐ  ． [38;5;2m                     [38;5;8m |[39m▄▄▁▄▄▁▄▄▂[38;5;2m░░░░░░░[38;5;8m | 00fa [39mａ ｐ ﹍ ｉ ｔ ﹍ ｕ ｐ  ． [38;5;2m                     [38;5;8m |[39m▄▄▁▄▄▁▄▄▂[38;5;2m░░░░░░░[39m
